docs/current_docs/manuals/developer/cookbook.mdx

```diff
@@ -130,14 +130,46 @@ dagger call github-api \
     --token=env:GITHUB_API_TOKEN
```

Comment:
I was thinking about slightly modifying this pattern of example usage to also include a brief one-liner for each case. See below.

Example usage:

Retrieve a list of issues for the `dagger/dagger` repository, reading the GitHub API token from the host environment variable `GITHUB_API_TOKEN`:

```shell
dagger call github-api \
    --endpoint=https://api.github.com/repos/dagger/dagger/issues \
    --token=env:GITHUB_API_TOKEN
```

Retrieve a list of issues for the `dagger/dagger` repository, reading the GitHub API token from the host file `./github.txt`:

```
```

Retrieve a list of issues for the `dagger/dagger` repository, reading the GitHub API token from the output of the `gh auth token` command on the host:

```
```

Reply:
I guess it depends if we're trying to have thorough documentation for the code **and** CLI in the cookbook. I was under the impression it's focused on code

Reply:
No, we definitely want to keep the focus on the code. The idea here was not to document all possible CLI examples in detail, but just to add a brief context sentence to each example. This is so that the reader doesn't need to guess what the CLI example does, or to avoid having to copy the function and run it to know what each example does.